src/views/mfa-verify/PasswordForm.js

```diff
@@ -15,7 +15,7 @@ define(['okta', './inputs/ShowPasswordBox'], function (Okta, PasswordBox) {
   var _ = Okta._;
 
   return Okta.Form.extend({
-    className: 'mfa-verify-question',
+    className: 'mfa-verify-password',
```

Comment:
This is only used in tests.. I didnt have a test using it because it is used in tests that change the factor using the dropdown